docs/data-sources/sombra.md

```diff
@@ -0,0 +1,31 @@
+---
+page_title: "transcend_sombra Data Source - terraform-provider-transcend"
+subcategory: ""
+description: |-
+  
+---
+
+# transcend_sombra (Data Source)
+
+
+
+## Example Usage
+
+You can search for a sombra id by the sombra url, which can be retrieved from the TRANSCEND INGRESS URL column in the admin dashboard under Infrastructure > Sombra. 
```

Comment:
Does this mean that the customer can simply set the data URL and we will resolve the ID for them..? or do they need to run a go script to find that ID and then set it in their terraform variable?

Reply:
Yes, in the customer's config file, main.tf, they can just include their sombra url and we will resolve the ID for them. Maybe my wording in the doc is off? I'll try to come up with another option.

Reply:
I think it's a bit confusing bc it sounds like the user need to "search" actively for the ID, when in reality, all we ned from them is the URL?

Reply:
Yeah this wording is a bit confusing -- maybe we can add a the ID to copy on the AD directly? Should be straightforward to add

Reply:
I can go either way, changing to wording verses adding the sombra ID to the AD for users to copy. Is there any reason not to have the sombraId on the AD?

As for copy, I was thinking something like, "Sombra Ids can be resolved through entry of a sombra url, which can be found in the admin dashboard."? I suck at writing...

Reply:
Makes sense to just say that the url can be used. I can't think of any other reason to reveal the sombra Id. Here's another attempt at wording:

`A sombra ID can be resolved through entry of the sombra url, which can be found in Admin Dashboard. Use the URL in the TRANSCEND INGRESS URL column in the Sombra table found under Infrastructure.`

Reply:
I just wouldn't mention sombra ID and would only allow customers to use/set the sombra via URL

Reply:
`Sombra can be set via the sombra url, which can be found in Admin Dashboard. Use the URL in the TRANSCEND INGRESS URL column in the Sombra table found under Infrastructure.`?

Reply:
> i feel like the user doesn't need to know about sombra IDs

We kinda have precedent for this, as we make users copy things like Enricher Ids, etc. from the AD when using the GQL API. In fact, Mike just very recently added a PR to add a copy button to the Enrichers settings.

But yeah, Sombra URL is also cool, and easier to understand. I would link to the appropriate [page directly](https://app.transcend.io/infrastructure/sombra/sombras) to  make it easier for document readers